src/components/EventBanner/styles.js

```diff
@@ -0,0 +1,55 @@
+const styles = (theme) => ({
+  eventCard: {
+    position: 'relative',
+    borderRadius: '5px',
```

Comment:
set the `borderRadius: 0`. you can see the corners of each card:

![image](https://user-images.githubusercontent.com/4350404/46845129-4bb2cc00-ce04-11e8-954a-01eae30b5dbe.png)